io.github.ja2-stracciatella.appdata.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>io.github.ja2-stracciatella</id>
```

Comment:
this app-id doesn't meet the requirement. It should have at least 4 components. (ie `io.github` (counts for 2), organization and app).

Reply:
but then the documentation say to replace `-` with `_`.

Reply:
Which documentation?  From [App requirements](https://github.com/flathub/flathub/wiki/App-Requirements#application-id) I came to [Flatpak IDs](https://docs.flatpak.org/en/latest/conventions.html#application-ids), which lists a few examples

Reply:
It says the name should be a valid bus name, but DBus bus names may contain `-`, it's just discouraged.  Where would I find the recommendation to replace the dash with an underscore?

That aside, the proper ID would then be `io.github.ja2_stracciatella.JA2Stracciatella`?

Reply:
And by the way, where do I actually find documentation that the current ID is invalid?  Flatpak itself didn't reject it, and [its documentation](https://docs.flatpak.org/en/latest/conventions.html#application-ids) just says that the current ID would be problematic but not that it violates some requirement.

Reply:
> Which documentation? From [App requirements](https://github.com/flathub/flathub/wiki/App-Requirements#application-id) I came to [Flatpak IDs](https://docs.flatpak.org/en/latest/conventions.html#application-ids), which lists a few examples but no guidelines.

Flatpak's:
https://docs.flatpak.org/en/latest/conventions.html#application-ids

Reply:
Oh dear, I have overlooked this sentence. I'm sorry for the noise.

I'll fix the ID 🙂